src/pages/trees/[treeid].js

```diff
@@ -284,7 +297,7 @@ export default function Tree({
                   mt: 2,
                 }}
               >
-                <Badges tokenId={tree.token_id} verified={tree.approved} />
+                <Badges content={badgesContent} />
```

Comment:
@khalatevarun can change the interface to something like this:
<Badge color="primary" name="xxxx" />
Don't use array

Reply:
okay

Reply:
I made some suggestions on #1643

Reply:
@khalatevarun I think the array one is harder to read. The code is scattered in places. And the array component is more complicated than single component, and hard to reuse

Reply:
Okay, will simply render Bage one after the another

Reply:
I suggested some things [here](https://github.com/Greenstand/treetracker-web-map-client/pull/1633#issuecomment-1598464948) @dadiorchen